docs/development/documentation_guidelines.rst

```diff
@@ -83,20 +108,14 @@ replace the existing screenshots. Stage for commit any screenshots you wish to
 update. If you wish to update all screenshots, simply stage for commit all
 changed files in that directory.
 
-Integration with Read the Docs
-------------------------------
 
-.. include:: ../includes/docs-branches.txt
+Style Guide
+-----------
 
-Our documentation is built and hosted by `Read the Docs`_ and is available at
-https://docs.securedrop.org. We use a `webhook`_ to rebuild the documentation
-automatically when commits get pushed to the branch.
+A quick reference on writting in `ReStructuredText`_ (ReST)
```

Comment:
I would suggest to make this a full sentence, e.g., "Please see the reStructuredText Primer[link] by the Sphinx project for an introduction to the markup language used for the documentation." or similar Also typo: `writting`